docs/guide/basic-usage.md

```diff
@@ -40,4 +40,139 @@ $query->select(['name', 'status'])
 // execute the query
 $rows = $query->all();
 ```
+## Aggregation
```

Comment:
Aggregation topic should not belong to `basic-usage` - it is an advanced usage and should be placed at separated file named `usage-aggregation.md`.